jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/BufferUtilTest.java

```diff
@@ -349,26 +344,4 @@ private void testWriteToWithBufferThatDoesNotExposeArray(int capacity) throws IO
         BufferUtil.writeTo(buffer.asReadOnlyBuffer(), out);
         assertThat("Bytes in out equal bytes in buffer", Arrays.equals(bytes, out.toByteArray()), is(true));
     }
-
-    @Test
-    public void testToMappedBufferResource() throws Exception
```

Comment:
Restore if the methods in `BufferUtil` are restored.

Reply:
You're right; reverted.